apps/valkyrie/mix.exs

```diff
@@ -39,7 +39,7 @@ defmodule Valkyrie.MixProject do
       {:credo, "~> 1.0", only: [:dev, :test], runtime: false},
       {:dead_letter, in_umbrella: true},
       {:distillery, "~> 2.1"},
-      {:divo, "~> 1.1", only: [:dev, :test, :integration]},
```

Comment:
same as alchemist, missing mix increment of app version number. since divo was updated in valkyrie, valkyrie needs a new mix version